tests/src/test/scala/slinky/core/ExternalComponentTest.scala

```diff
@@ -65,9 +65,11 @@ class ExternalComponentTest extends FunSuite {
   test("Cannot reuse half-built external component") {
     val halfBuilt = ExternalDivWithProps(id = "test")
     val fullyBuilt: ReactElement = halfBuilt.withKey("abc")
+    println(fullyBuilt)
```

Comment:
Similar `println`'s here. If this is to just make the unused variable warning go away, perhaps do something like a simple `assert(fullyBuild != null)`?